src/main/java/com/webonise/todoapp/controller/TodoController.java

```diff
@@ -0,0 +1,42 @@
+package com.webonise.todoapp.controller;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import com.webonise.todoapp.model.Todo;
+import com.webonise.todoapp.service.TodoService;
+
+@RestController
+@CrossOrigin(origins = "*", allowedHeaders = "*")
```

Comment:
Any specific reason to use this?

Reply:
I got the cors policy error thats why i have added that to allow request from any origin and for all headers.